js/api.md

```diff
@@ -983,7 +983,7 @@ const client = new AWSAppSyncClient({
 
 ##### Cognito User Pools Auth
 
-Amazon Cognito User Pools is the most common service to use with AppSync when adding user Sign-Up and Sign-In to your application. If your application needs to interact with other AWS services besides AppSync, such as S3, you will need to use IAM credentials with Cognito Identity Pools. The Amplify CLI can automatically configure this for you when running `amplify add auth` and can also automatically federate User Pools with Identity Pools. This allows you to have both User Pool credentials for AppSync and AWS credentials for S3. You can then use the `AWSMobileClient` for automatic credentials refresh [as outlined in the authentication section](./authentication). For manual configuration, add the following snippet to your `aws-exports.js` file:
+Amazon Cognito User Pools is the most common service to use with AppSync when adding user Sign-Up and Sign-In to your application. If your application needs to interact with other AWS services besides AppSync, such as S3, you will need to use IAM credentials with Cognito Identity Pools. The Amplify CLI can automatically configure this for you when running `amplify add auth` and can also automatically federate User Pools with Identity Pools. This allows you to have both User Pool credentials for AppSync and AWS credentials for S3. You can then use the `Auth` category for automatic credentials refresh [as outlined in the authentication section](./authentication). For manual configuration, add the following snippet to your `aws-exports.js` file:
```

Comment:
this may cause confusion: "and can also automatically federate User Pools with Identity Pools"

not sure if that means i will get that with `amplify add auth` or if it's something else i need to do

Reply:
Let's leave that as-is since it's not the thing that was updated and hasn't caused confusion yet.